pkg/logging/multihandler.go

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package logging
+
+import (
+	"context"
+	"log/slog"
+
+	"github.com/cilium/cilium/pkg/lock"
+)
+
+func NewMultiSlogHandler(handler slog.Handler) *multiSlogHandler {
```

Comment:
nit for readability: Comment above constructor
```suggestion
// NewMultiSlogHandler creates an slog.Handler that supports multiple
// underlying handlers, such as to output text and json.
func NewMultiSlogHandler(handler slog.Handler) *multiSlogHandler {
```

This is what the MultiHandler is for right?